analyze/dev_linux.go

```diff
@@ -39,7 +39,10 @@ func GetDevicesInfo() []*Device {
 				Size:       info.Bsize * int64(info.Blocks),
 				Free:       info.Bsize * int64(info.Bavail),
 			}
-			devices = append(devices, device)
+			
+			if !(strings.Contains(parts[1], "/snap/")){
+				devices = append(devices, device)
+			}
```

Comment:
Thanks for your contribution!

I think it would be a bit better if the check was before the syscall (and will do `continue`). Also please run go fmt before submiting (there should be one space before the opening curly brace).

Reply:
Good point, I missed that. Also thank you for the tips, I wasn't aware of `go fmt`. Is it OK now?

Reply:
Codecov will be a bit harder. I will think about it later.

It's ok, just one last bit: you added `main` binary by mistake. Remove it please :)

Reply:
Ah.. Sorry for the mess, done.